docs/instructor-setup.md

```diff
@@ -0,0 +1,211 @@
+update endpoint.
+
+## Custom Signet
+This workshop uses a custom `signet` for the following reasons:
+
+- We want a confined hashrate environment, so `mainnet`, `testnet3`, `testnet4` and the public `signet` are ill suited.
+- `regtest` is too isolated and requires manual block generation, which is not practical for a collaborative workshop setting.
+- We will mine on a custom `signet` that does not require coinbase signatures.
+- This way, we can deploy pools + hashers and emulate a confined hashrate environment.
+
+Participants will connect to this Genesis node to sync their blocks.
+
+### Genesis Node
+The instructor can use the existing Genesis node hosted on the SRI VM, or spin up their own. The
+Genesis node should be configured via the [materials/signet-genesis-node.sh](https://github.com/stratum-mining/sv2-workshop/blob/main/materials/signet-genesis-node.sh) script which:
+
+* Deploys a local signet using the [`bitcoin.conf`](https://github.com/stratum-mining/sv2-workshop/blob/main/materials/genesis-bitcoin.conf).
+* Mines 16 blocks as bootstrapping for the SRI pool.
+
+Before executing the script, ensure the following environment variables are defined:
+
+```sh
+$ export BITCOIND=$HOME/bitcoin/src/bitcoind
+$ export BITCOIN_CLI=$HOME/bitcoin/src/bitcoin-cli
+$ export BITCOIN_UTIL=$HOME/bitcoin/src/bitcoin-util
+$ export MINER=$HOME/bitcoin/contrib/signet/miner
+$ export BITCOIN_DATA_DIR=$HOME/.bitcoin
+```
+
+#### SRI Node
+A Genesis node that is publicly accessible is needed for participants to sync their Bitcoin nodes.
+This can be set up by the instructor or use the existing SRI VM node.
+
+If using the SRI hosted Genesis node, verify it is running by remoting into the SRI VM and finding
+its process:
+
+```sh
+ps -ef | grep -i bitcoind
+> sri 3935787 1 0 Jun29 ? 01:19:13 /home/sri/btc_prague_workshop/bitcoin/src/bitcoind -signet -datadir=/home/sri/btc_prague_workshop/bitcoin_data_dir/ -fallbackfee=0.01 -daemon -sv2 -sv2port=38442
+```
+
+If spinning up a new node, see the instructions to install `bitcoin-core` in the Block Explorer
+section below.
+
+## Block Explorer
+A block explorer is needed to display participants' mined blocks on the custom `signet`. A custom
+`signet` Bitcoin node, [`electrs`](https://github.com/romanz/electrs), and
+[`mempool.space`](https://github.com/mempool/mempool) is used for this purpose.
+
+> Note: The steps for deploying a local `mempool.space` have not yet been automated and will need to
+be performed manually.
+
+> Note: The Genesis node can also be used for this purpose. A second Bitcoin node for the block
+explorer only is needed if the instructor is running the block explorer on another machine (like
+their local machine).
+
+### Custom Signet `bitcoin-core`
+
+#### Install
+There are two ways to install the required `bitcoin-core` fork:
+
+1. Release Binary: [Plebhash's fork of Sjors's sv2-tp-0.1.3 tag](https://github.com/plebhash/bitcoin/releases/tag/btc-prague).
```

Comment:
I only created my fork to provide for `aarch64-apple` binary

if it's recommended to build from source in this case, it's better to just send the audience towards Sjors release instead of mine